.travis.yml

```diff
@@ -42,7 +42,7 @@ env:
 
         # List other runtime dependencies for the package that are available as
         # pip packages here.
-        - PIP_DEPENDENCIES='sphinx_rtd_theme sphinx_automodapi specutils==0.2.2 qtpy'
+        - PIP_DEPENDENCIES='sphinx_rtd_theme sphinx_automodapi specutils==0.2.2 qtpy git+https://github.com/astropy/regions.git'
```

Comment:
Can you maybe leave a comment here that says *why* this is the dev version of regions?  I.e. "need the work in master as of Sep 11, 2018 which has not yet been released".

Otherwise I fear we will forget why it's a dev version and constantly be saying "hmm, not sure it's safe to switch to a release"...